.dependabot/config.yml

```diff
@@ -1,29 +1,13 @@
 # Reference: https://dependabot.com/docs/config-file/
 version: 1
 update_configs:
-  # This configures dependency updates for one package manager. In some
-  # projects, such as warehouse, where we have Ruby and Python, there can be
-  # separate package_manager entries.
   - package_manager: "ruby:bundler"
     directory: "/"
-    update_schedule: "weekly"
-
-    default_labels:
-      - "dependencies"
-      - "Needs QA"
-
-    # Dependabot will use a repository's default branch. This will override
-    # that.
-    # target_branch: "master"
-
-    allowed_updates:
-      - match:
-          dependency_type: "direct"
-
+    update_schedule: "live"
```

Comment:
I don't see an advantage to doing these weekly, since the main reason for that was to isolate our Buildkite Dependabot builds but this repo doesn't use Buildkite. Open to pushback though!